schema.sql

```diff
@@ -0,0 +1,98 @@
+CREATE TABLE region (
+  id uuid PRIMARY KEY,
+  climate_type text NOT NULL,
+  title text NOT NULL,
+  description text,
+  image_thumb uuid,
+  image_cover uuid
+);
+
+CREATE TABLE vineyard (
+  id uuid PRIMARY KEY,
+  name text NOT NULL,
+  location point,
+  description text,
+  picture_thumb uuid,
+  picture_cover uuid,
+  region_id uuid REFERENCES region(id)
+);
+
+CREATE TABLE wine (
+  id uuid PRIMARY KEY,
+  name text NOT NULL,
+  vintage integer NOT NULL,
+  vineyard_id uuid REFERENCES vineyard(id),
+  alcohol real NOT NULL,
+  sugar real NOT NULL,
+  acid real NOT NULL,
+  brix real NOT NULL,
+  body real NOT NULL,
+  color text NOT NULL,
+  is_red boolean NOT NULL,
+  tannin real NOT NULL,
+  fruity boolean NOT NULL,
+  spicy boolean NOT NULL,
+  tart boolean NOT NULL,
+  label_description text,
+  price money NOT NULL
+);
+
+CREATE TABLE grape (
+  id uuid PRIMARY KEY,
+  name text NOT NULL,
+  description text,
+  image_thumb uuid,
+  berry_skin_color text,
+  also_known_as text,
+  idea_soil text,
+  general text,
+  cool text,
+  mid text,
+  hot text
+);
+
+CREATE TABLE grape_component (
+  id uuid PRIMARY KEY,
+  wine_id uuid REFERENCES wine(id),
+  grape_id uuid REFERENCES grape(id),
+  percent real
+);
+
+CREATE TABLE foods (
+  id uuid PRIMARY KEY,
+  name text NOT NULL,
+  picture uuid,
+  fatty boolean NOT NULL,
+  sweet boolean NOT NULL,
+  dense boolean NOT NULL,
+  starchy boolean NOT NULL,
+  delicate boolean NOT NULL,
+  strong boolean NOT NULL,
+  description text
+);
+
+CREATE TABLE users (
+  id uuid PRIMARY KEY,
+  name text NOT NULL,
+  email text NOT NULL,
+  password text,
+  likes bigint NOT NULL DEFAULT 0,
+  bio text
+);
```

Comment:
This is definitely my fault for forgetting this, but we're going to need some sort of way to distringuish regular users from site admins. E.g. only site admins can add wine regions. I think a role bit/smallint might be good, otherwise just an isAdmin bool should be fine.